sdk-core/src/main/java/com/reach5/identity/sdk/core/models/Profile.kt

```diff
@@ -28,11 +32,10 @@ data class Profile(
     // TODO better type for address
     val addresses: List<ProfileAddress>? = null,
     @SerializedName("custom_fields")
-    val customFields: Map<String, Any>? = null
-) {
+    val customFields: Map<String, @RawValue Any>? = null
```

Comment:
What does @RawValue do?

Reply:
@rawvalue is an annotation required by @Parcelize to serialize `Any`
https://stackoverflow.com/questions/49606163/rawvalue-annotation-is-not-applicable-to-target-value-parameter
If you have a better idea, I am open to hear it 😄